appveyor.yml

```diff
@@ -32,44 +32,53 @@
     - '%AppData%\npm-cache'
 
   environment:
+    # https://github.com/jasongin/nvs/blob/master/doc/CI.md
+    NVS_VERSION: 1.4.0
     SKIP_SASS_BINARY_DOWNLOAD_FOR_CI: true
     matrix:
-      - nodejs_version: 0.10
+      - NODEJS_VERSION: node/0.10
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 0.12
+      - NODEJS_VERSION: node/0.12
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 1.0
+      - NODEJS_VERSION: iojs/1
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 1
+      - NODEJS_VERSION: iojs/2
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 2
+      - NODEJS_VERSION: iojs/3
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 3
+      - NODEJS_VERSION: node/4
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 4
+      - NODEJS_VERSION: node/5
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 5
-        GYP_MSVS_VERSION: 2013
-        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 6
+      - NODEJS_VERSION: node/6
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      - nodejs_version: 7
+      - NODEJS_VERSION: node/7
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      - nodejs_version: 8
+      - NODEJS_VERSION: node/8
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - NODEJS_VERSION: chakracore/8.1.2
```

Comment:
This raises the question of what kind of ABI guarantees we get with CharkraCore. 

Can we continue to rely on process.modules.version as the ABI version?
Is CharkraCore supported by N-API?
What's the estimated time line on WASM support in CharkraCore?